packages/augur-ui/src/modules/contracts/actions/contractCalls.ts

```diff
@@ -503,6 +573,51 @@ export interface CreateNewMarketParams {
   template?: ExtraInfoTemplate;
 }
 
+export function createMarketEstimateGas(
+  newMarket: CreateNewMarketParams,
+  isRetry: Boolean
+) {
+  const params = constructMarketParams(newMarket, isRetry) as any;
+  const Augur = augurSdk.get();
+  const { universe } = Augur.contracts;
+
+  switch (newMarket.marketType) {
+    case SCALAR: {
+      return universe.createScalarMarket_estimateGas(
+        params.endTime,
+        params.feePerCashInAttoCash,
+        '0x0000000000000000000000000000000000000000',
+        params.affiliateFeeDivisor,
+        '0x0000000000000000000000000000000000000001',
+        params.prices,
+        params.numTicks,
+        params.extraInfo
+      );
+    }
+    case CATEGORICAL: {
+      return universe.createCategoricalMarket_estimateGas(
+        params.endTime,
+        params.feePerCashInAttoCash,
+        '0x0000000000000000000000000000000000000000',
```

Comment:
There us a constant NULL_ADDRESS for all zero address. 
Why use different address for other address param?

Reply:
Will update constant 👍
I was getting contract errors using NULL_ADDRESS as the address param